lang_python/parsing/parser_python.mly

```diff
@@ -689,12 +689,20 @@ string:
 
 interpolated:
   | FSTRING_STRING { Str $1 }
-  | FSTRING_LBRACE test RBRACE { $2 }
-  | FSTRING_LBRACE test COLON format_specifier RBRACE 
+  | FSTRING_LBRACE interpolant RBRACE { $2 }
+  | FSTRING_LBRACE interpolant COLON format_specifier RBRACE
      { InterpolatedString ($2::mk_str $3::$4) }
-  | FSTRING_LBRACE test BANG format_specifier RBRACE 
+  | FSTRING_LBRACE interpolant BANG format_specifier RBRACE
      { InterpolatedString ($2::mk_str $3::$4) }
 
+interpolant:
```

Comment:
Great!
Maybe you can say that fstrings are actually not specified in the grammar, but 'testlist' looks like the good candidate 
so it can parse tuples, not just basic expressions (like with the 'test' rule)
See https://docs.python.org/3/reference/grammar.html

Reply:
👍